lib/compilers/postcss-compiler.js

```diff
@@ -0,0 +1,28 @@
+const postcss = require('postcss')
+const postcssrc = require('postcss-load-config')
+const ctx = { parser: true, map: 'inline' }
+const { plugins } = postcssrc.sync(ctx)
+
+const throwError = require('../throw-error')
+
+module.exports = (content) => {
+  let css = null
+
+  postcss(plugins)
+    .process(content, {
+      from: undefined
+    })
+    .then(result => {
+      css = result.css || ''
+    })
+    .catch((e) => {
+      css = ''
+      throwError(e)
```

Comment:
Instead of throwing an error, can you log an error if the hideStyleWarn is false, like this: https://github.com/mrvasia/vue-jest/blob/dd57354402945c796466d33dc95d4d2d7f73babe/lib/compilers/sass-compiler.js#L29

Reply:
fixed